### DISCUSSION TITLE
ChatGPT注册与使用.md

### DISCUSSION THREAD

**Discussion by giscus** [xdd1997/docsify#24] [General]
# ChatGPT注册与使用.md

https://www.20161101.xyz/docsify/#/md_File/%E6%9C%AA%E5%88%86%E7%B1%BB/003ChatGPT%E6%B3%A8%E5%86%8C%E4%B8%8E%E4%BD%BF%E7%94%A8

**Comment by xdd1997** [comment]
测试一下

**Reply by xdd1997** [reply]
002

**Comment by xdd1997** [comment]
测试2
